### ISSUE TITLE
GraphicsWindow.DrawImage doesn't show the image

### ISSUE THREAD

**Issue by nonkit** [sb/smallbasic-editor#58]
Following program doesn't show the image.

```
path = "http://www.nonkit.com/smallbasic.files/Turtle.png"
GraphicsWindow.DrawImage(path, 10, 10)
```

**Comment by OmarTawfik** [comment]
@nonkit mmm, from the docs in the old editor I found, I was under the impression that `GraphicsWindow.DrawImage` takes an image name that was loaded using `ImageList.LoadImage()` method. Maybe we need to support urls and file paths here as well.

**Comment by estberg** [comment]
In the [current online editor](https://smallbasic-publicwebsite.azurewebsites.net/Program/Editor.aspx), if we use [the code](https://smallbasic-publicwebsite.azurewebsites.net/Program/view.aspx?KWF239) that nonkit included, we can get an image to come up without using `ImageList.LoadImage()`.  It isn't in [Super Basic](http://superbasic-v2.azurewebsites.net/) yet though. Since it works on the old version should it be implemented for the new version?

**Comment by OmarTawfik** [comment]
I think so. We should definitely implement it (would need to see of CORS settings would allow us to do so).
A caveat would be file paths. If the user would be allowed to also load local files through this API, then we would need to restrict this to the SuperBasic desktop editor only.

**Comment by nonkit** [comment]
Hi everyone, I could confirm #58 fixed with Chrome.  But Edge couldn't show the image with the code above.   Is it another issue?  If so, I will close this issue.
cc: @OmarTawfik @estberg @shouryaj @EdPrice-MSFT @xuachen

**Comment by alxnull** [comment]
Added to @nonkit's observations, the problem also persists in Firefox for me. This seems to be caused by Firefox requiring the [`width` and `height` attributes of the image element](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/image#Specific_attributes).

**Comment by Caseysch** [comment]
Now working on Chrome, Edge, and Firefox

**Comment by nonkit** [comment]
I also confirmed #58 fixed both on Edge 84.0.522.52 (Official build) (64-bit) and Chrome 84.0.4147.105 (Official Build) (64-bit). Thanks!
